src/components/buttons/AddRemoveButton.tsx

```diff
@@ -1,36 +0,0 @@
-import React from "react";
-
-interface AddRemoveButtonProps {
-  children: string;
```

Comment:
I think if you want to make this more flexible (as in adding an icon on the button), you should change the children type to `React.JSXElement` I think.